tests/Feature/Http/Controllers/Auth/AuthenticationTest.php

```diff
@@ -38,7 +38,7 @@ public function testLoginScreenCanBeRendered(): void
     public function testUsersCanAuthenticateUsingTheLoginScreen(): void
     {
         // Given
-        $user = User::factory()->create();
+        $user = AuthorizedUser::factory()->create();
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar $authorizedUser .

```suggestion
        $authorizedUser = AuthorizedUser::factory()->create();
````